.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Question
+about: Ask a question about this project.
+title: ''
+labels: question
+assignees: ''
+
+---
+
+<!--
+Please search existing issues to avoid creating duplicates.
+Also consider checking the issues over at the [vscode repository](https://github.com/microsoft/vscode-python/issues) as well. 
```

Comment:
Wrong repo.

Reply:
I'm linking to the vscode repo because some things might be answered there

Reply:
Oh, I see. Then change the text in the link to "VS Code Python extension repo" instead (vscode is its own thing).